code/datums/elements/weapon_description.dm

```diff
@@ -0,0 +1,186 @@
+ *
+ * Arguments:
+ * 	* source - Object being examined, cast into an item variable
+ *  * user - Unused
+ *  * examine_texts - The output text list of the original examine function
+ */
+/datum/element/weapon_description/proc/warning_label(obj/item/item, mob/user, list/examine_texts)
+	SIGNAL_HANDLER
+
+	if(item.force >= 5 || item.throwforce >= 5 || item.override_notes || item.offensive_notes) /// Only show this tag for items that could feasibly be weapons, shields, or those that have special notes
+		examine_texts += "<span class='notice'>It appears to have an ever-updating bluespace <a href='?src=[REF(item)];examine=1'>warning label.</a></span>"
+
+/**
+ *
+ * Details the stats of the examined weapon
+ *
+ * This function is called when the user clicks the hyperlink provided by
+ * warning_label(). It calls build_label_text() and outputs its return value to the user
+ *
+ * Arguments:
+ *  * source - Object being examined, sent to build_label_text()
+ *  * href-list - List provided by the href of input values, used to know what hyperlinked action is being attempted
+ */
+
+/datum/element/weapon_description/proc/topic_handler(atom/source, user, href_list)
+	SIGNAL_HANDLER
+
+	if(href_list["examine"])
+		to_chat(user, "<span class='notice'>[build_label_text(source)]</span>")
+
+/**
+ *
+ * Compiles a warning label detailing various statistics of the examined weapon
+ *
+ * This function is called by the "examine" function of Topic(), and compiles a number of relevant
+ * weapon stats into a message that is then shown to the user
+ * Arguments:
+ *  * source - The object whose stats are being examined
+ */
+/datum/element/weapon_description/proc/build_label_text(obj/item/source)
+	var/list/readout = list("") // Readout is used to store the text block output to the user so it all can be sent in one message
+
+	// Meaningless flavor text. The number of crimes is constantly changing because of the complex Nanotrasen legal system and the esoteric nature of time itself!
+	readout += "<span class='warning'>WARNING:</span> This item has been marked as dangerous by the NT legal team because of its use in <span class='warning'>[rand(2,99)] [crimes[rand(1, crimes.len)]]</span> in the past hour.\n"
+
+	// Doesn't show the base notes for items that have the override notes variable set to true
+	if(!source.override_notes)
+		// Make sure not to divide by 0 on accident
+		if(source.force > 0)
+			readout += "Our extensive research has shown that it takes a mere <span class='warning'>[round((100 / source.force), 0.1)]</span> hits to down [victims[rand(1, victims.len)]] with no armor."
+		else
+			readout += "Our extensive research found that you couldn't beat yourself to death with this if you tried."
+
+		if(source.throwforce > 0)
+			readout += "If you decide to throw this object instead, one will take <span class='warning'>[round((100 / source.throwforce), 0.1)]</span> hits before collapsing."
```

Comment:
```suggestion
			readout += "If you decide to throw this object instead, one will take <span class='warning'>[round((100 / source.throwforce), 0.1)]</span> hit\s before collapsing."
```

Otherwise "1 hits before collapsing"